datasets/noun_verb/noun_verb.py

```diff
@@ -0,0 +1,124 @@
+# coding=utf-8
+# Copyright 2020 HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+import conllu
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{NOUNVERB,
+  title = {A Challenge Set and Methods for Noun-Verb Ambiguity},
+  author = {Ali Elkahky and Kellie Webster and Daniel Andor and Emily Pitler},
+  booktitle = {Proceedings of EMNLP},
+  year = {2018}
+}
+"""
+
+_DESCRIPTION = """\
+This dataset contains naturally-occurring English sentences that feature non-trivial noun-verb ambiguity.
+
+English part-of-speech taggers regularly make egregious errors related to noun-verb ambiguity, despite having achieved 97%+ accuracy on the WSJ Penn Treebank since 2002. These mistakes have been difficult to quantify and make taggers less useful to downstream tasks such as translation and text-to-speech synthesis.
+
+Below are some examples from the dataset:
+
+- Certain insects can damage plumerias, such as mites, flies, or aphids. NOUN
+- Mark which area you want to distress. VERB
+
+All tested existing part-of-speech taggers mistag both of these examples, tagging flies as a verb and Mark as a noun.
+"""
+
+_HOMEPAGE = "https://github.com/google-research-datasets/noun-verb"
+
+_TRAIN_URL = "https://raw.githubusercontent.com/google-research-datasets/noun-verb/master/train.conll"
+_DEV_URL = "https://raw.githubusercontent.com/google-research-datasets/noun-verb/master/dev.conll"
+_TEST_URL = "https://raw.githubusercontent.com/google-research-datasets/noun-verb/master/test.conll"
+
+
+class NounVerb(datasets.GeneratorBasedBuilder):
+    VERSION = datasets.Version("1.0.0")
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("int32"),
+                    "token_ids": datasets.Sequence(datasets.Value("int32")),
+                    "tokens": datasets.Sequence(datasets.Value("string")),
+                    "lemmas": datasets.Sequence(datasets.Value("string")),
+                    "upos_tags": datasets.Sequence(datasets.Value("string")),
+                    "xpos_tags": datasets.Sequence(datasets.Value("string")),
```

Comment:
Can you use the ClassLabel feature type for those two fields please ?
You can take a look at an example of usage in [conll2003](https://github.com/huggingface/datasets/blob/master/datasets/conll2003/conll2003.py)

Reply:
Hi @abhishekkrthakur what do you think about using class label ?